lib/benchmark_driver/ruby_interface.rb

```diff
@@ -66,11 +61,19 @@ def compare!
     end
 
     def rbenv(*versions)
-      versions.each do |version|
-        @executables << BenchmarkDriver::Rbenv.parse_spec(version)
+      @config.executables.replace versions.map do |version|
+        BenchmarkDriver::Rbenv.parse_spec(version)
       end
     end
 
+    def reset_executables
```

Comment:
I don't think this is needed, because of the current `#run` behavior. It's empty by default and so of course the reset is not needed by default, and I couldn't think of any motivation to dynamically reset the executables after some unnecessary executables are specified